tests/testthat/test-evaluate-clusters.R

```diff
@@ -72,20 +119,65 @@ test_that("calculate_stability works as expected with different replicates", {
 test_that("calculate_stability works as expected with object and pc_name", {
   reducedDimNames(sce) <- "my_pca"
 
-  # note that we suppress warnings since this calculation done on fake
-  # test data gives expected warnings about ties during the ARI calculation.
-  suppressWarnings({
-    df <- calculate_stability(
-      sce,
-      cluster_df,
-      replicates = 2,
-      pc_name = "my_pca"
-    )
-  })
+  df <- calculate_stability(
+    sce,
+    cluster_df,
+    replicates = 2,
+    pc_name = "my_pca"
+  )
   expect_equal(nrow(df), 2)
 })
 
 
+test_that("calculate_stability warnings argument works", {
+  expect_silent({
+    df <- calculate_stability(test_mat, cluster_df)
+  })
+
+  # only run the next test on lower versions of bluster
+  skip_if(packageVersion("bluster") >= "1.16.0")
```

Comment:
Why? This comment should explain what has changed that the test is not required. But also, if this warning is gone in higher versions of `bluster`, is that only part of Bioconductor 3.20, or is there a version within Bioc3.19 that avoids the need for the warnings at all?